lib/Driver/Ldap.php

```diff
@@ -83,7 +83,7 @@ protected function _connect()
         }
 
         /* Set the LDAP referrals. */
-        if (!empty($this->_params['referrals'])) {
+        if (array_key_exists('referrals', $this->_params)) {
```

Comment:
Maybe this actually needs both checks? array_key_exists() and then also check if that array key's value is not empty?

Reply:
That line has already been changed some time ago.